modules/virt-creating-a-vm-from-a-boot-source.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc
+
+[id="virt-creating-a-vm-from-a-template-with-an-attached-boot-source_{context}"]
+= Creating a virtual machine from a template with an attached boot source
+
+After you add a boot source to a template, you can create a new virtual machine from the template.
+
+.Procedure
+
+. In the {product-title} web console, click *Workloads > Virtualization* in the side menu.
+. Select *Create Virtual Machine > New with Wizard*. The *Create Virtual Machine* wizard opens.
```

Comment:
These steps about virtual machine creation is not using the latest wizard on 2.6 UI.

Reply:
Changed step two to read:

"From the **Virtual Machines** tab or the **Templates** tab, click **Create** and select **Virtual Machine with Wizard**".

This matches the UI documented by @lmandavi in her latest UI PRs for CNV 2.6.

Reply:
@gouyang If you approve this change (latest test build is https://deploy-preview-28763--osdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.html) I will submit this for merge to master branch for CNV doc today, 2/17/21. Thanks, Bob

Reply:
Hi @gouyang - See test build: https://deploy-preview-28763--osdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.html

If you approve, we can merge to master today, 2/17/21. Thanks, Bob